src/api-reference/expense/expense-report/v4.comments.markdown

```diff
@@ -235,6 +235,240 @@ curl --location --request GET 'https://us.api.concursolutions.com/expensereports
     }
 ]
 ```
+## <a name="post-report-header-comments"></a>Post Report Header Comments
+
+Creates a new comment for the specified report.
+
+### Scopes
+
+`expense.report.read` - Refer to [Scope Usage](#scope-usage) for full details.
```

Comment:
expense.report.readwrite scope is used for POST API. Please refer https://github.concur.com/inception/expense-report-service/blob/develop/src/main/java/com/concur/expense/web/controllers/CommentController.java#L140